core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/ImageCatalogV4Endpoint.java

```diff
@@ -180,7 +180,7 @@ ImageV4Response getImageFromDefault(@PathParam("workspaceId") Long workspaceId,
             @PathParam("govCloud") boolean govCloud) throws Exception;
 
     @GET
-    @Path("image/type/{type}/provider/{provider}")
+    @Path("image/type/{type}/provider/{provider}/govCloud/{govCloud}")
```

Comment:
Sanity check: will this have any effects on other clients calling this endpoint, e.g. CDP CLI?

Reply:
I just removed this. Seems no client using it. This endpoint will be not govcloud compatible